library/src/main/java/com/owncloud/android/lib/resources/assistant/GetTaskListRemoteOperation.kt

```diff
@@ -18,14 +18,14 @@ import com.owncloud.android.lib.resources.OCSRemoteOperation
 import com.owncloud.android.lib.resources.assistant.model.TaskList
 import org.apache.commons.httpclient.HttpStatus
 
-class GetTaskListRemoteOperation(private val appId: String) : OCSRemoteOperation<TaskList>() {
+class GetTaskListRemoteOperation(private val taskType: String) : OCSRemoteOperation<TaskList>() {
```

Comment:
If we have a data class TaskType, we can use it here, right?

Reply:
https://github.com/nextcloud/android-library/pull/1599/commits/fab1e27e64a8636a09bd6ce5ac36c41e01965905